code/datums/elements/crusher_loot.dm

```diff
@@ -0,0 +1,41 @@
+/**
+ * Crusher Loot; which makes the attached mob drop a crusher trophy of some type if the majority damage was from a crusher!
+ *
+ * Used for all the mobs droppin' crusher trophies
+ */
+/datum/element/crusher_loot
+	element_flags = ELEMENT_BESPOKE|ELEMENT_DETACH
+	id_arg_index = 2
+	/// Path of the trophy dropped
+	var/trophy_type
+	/// chance to drop the trophy if all the damage the mob took was crusher damage
```

Comment:
this comment is a little confusing, since it implies the trophy only drops if the mob only took crusher damage